app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -205,7 +205,10 @@ class KaigiAppMainNestedGraphStateHolder : MainNestedGraphStateHolder {
             Timetable -> mainNestedNavController.navigateTimetableScreen()
             About -> mainNestedNavController.navigateAboutScreen()
             FloorMap -> mainNestedNavController.navigateFloorMapScreen()
-            Contributor -> mainNestedNavController.navigate(contributorsScreenRoute)
+            Contributor -> mainNestedNavController.navigate(contributorsScreenRoute) {
+                launchSingleTop = true
+                restoreState = true
+            }
```

Comment:
It's different from other cases. Should it fix?

Reply:
This screen is created to investigate the Compose Multiplatform. so we are not using navigation for now. I think we can have it in androidMain though.